### PR TITLE
tor: update to version 0.4.8.9

### PR DESCRIPTION
**Pull request by svschmel**
A new (security) tor version has been released.

=> https://forum.torproject.org/t/security-release-0-4-8-9/10155

Works fine.
https://metrics.torproject.org/rs.html#details/AD0FCFD83EA4899DB34735E599701592DEA13690
https://metrics.torproject.org/rs.html#details/448F821108ED29AA408F281625F66BE1D4BC19F4

"gmake test" has one error.
No difference to version 0.4.8.8.
"results-all.master" has not changed.

```
svschmel@oibuild:~/oi-userland/components/network/tor/build/amd64$ cat test-suite.log 
===================================
   tor 0.4.8.9: ./test-suite.log
===================================

# TOTAL: 33
# PASS:  30
# SKIP:  2
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

SKIP: src/test/fuzz_static_testcases.sh
=======================================

You need to set TOR_FUZZ_CORPORA to point to a checkout of 
the 'fuzzing-corpora' repository.
SKIP src/test/fuzz_static_testcases.sh (exit status: 77)

SKIP: src/test/test_switch_id.sh
================================

This test only works when run as root. Skipping.
SKIP src/test/test_switch_id.sh (exit status: 77)

FAIL: src/test/test_bt.sh
=========================

OK
BAD

============================================================ T= 1699601306
Tor died: Caught signal 11
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'crash_handler+0x5f [0x40f45f]
/lib/amd64/libc.so.1'__sighndlr+0x6 [0x7fffa9402326]
/lib/amd64/libc.so.1'call_user_handler+0x1d1 [0x7fffa93f4f01]
/lib/amd64/libc.so.1'sigacthandler+0xee [0x7fffa93f51fe]
[0xffffffffffffffff]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'oh_what+0x25 [0x407295]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'a_tangled_web+0x26 [0x4072e6]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'we_weave+0x25 [0x407335]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'main+0xae [0x40740e]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'_start_crt+0x87 [0x407117]
/export/home/svschmel/oi-userland/components/network/tor/build/amd64/src/test/test-bt-cl'_start+0x18 [0x407078]
-158318
FAIL src/test/test_bt.sh (exit status: 1)
```

For this error i have opened the issue :
https://gitlab.torproject.org/tpo/core/tor/-/issues/40826

### CODE REVIEW

**Comment by AndWac** [comment]
We have gcc-13 as our new default compiler. pkg5 should be updated. That happened automatically when you run gmake publish.

**Comment by AndWac** [comment]
Ok, now please squash the commits.

**Comment by svschmel** [comment]
Sollte jetzt passen:

---
svschmel@oibuild:~/oi-userland/components/network/tor$ git rebase -i HEAD~2
Successfully rebased and updated refs/heads/tor.0.4.8.9.

svschmel@oibuild:~/oi-userland/components/network/tor$ git push -f origin tor.0.4.8.9
Enter passphrase for key '/export/home/svschmel/.ssh/id_rsa': 
Everything up-to-date

**Comment by AndWac** [comment]
There are still two commits (and its commit messages) in the PR. While Github squashes the commits automatically, it appends all commit messages resulting in unwanted redundant messages.

**Comment by svschmel** [comment]
Sorry.
I have done a mistake.
Now it should work.

File: components/network/tor/Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		tor
-COMPONENT_VERSION=	0.4.8.8
+COMPONENT_VERSION=	0.4.8.9
 COMPONENT_REVISION=	1
 COMPONENT_PROJECT_URL=	https://www.torproject.org
```
Comment (AndWac) [inline components/network/tor/Makefile:21]: Please remove COMPONENT_REVISION

**Comment by svschmel** [comment]
Please remove COMPONENT_REVISION => done

**Pull request merged**